tests/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -1759,6 +1759,18 @@ public void TestFillNulls()
             Assert.Equal(0, strColumn.NullCount);
             Assert.Equal("foo", strColumn[0]);
             Assert.Equal("foo", strColumn[1]);
+
+            // ArrowStringColumn (not inplace)
+            ArrowStringDataFrameColumn arrowColumn = CreateArrowStringColumn(3);
+            Assert.Equal(3, arrowColumn.Length);
+            Assert.Equal(1, arrowColumn.NullCount);
+            Assert.Equal(null, arrowColumn[1]);
+            ArrowStringDataFrameColumn arrowColumnFilled = arrowColumn.FillNulls("foo");
+            Assert.Equal(3, arrowColumn.Length);
+            Assert.Equal(1, arrowColumn.NullCount);
+            Assert.Equal(3, arrowColumnFilled.Length);
+            Assert.Equal(0, arrowColumnFilled.NullCount);
+            Assert.Equal("foo", arrowColumnFilled[1]);
```

Comment:
Can you also add the following 2 lines here?

`Assert.Equal(arrowColumn[0], arrowColumnFilled[0]);`
`Assert.Equal(arrowColumn[2], arrowColumnFilled[2]);`

Because we're manipulating the underlying buffers, it'd be a good idea to check that the 0th and 2nd values are the same before and after `FillNulls`.

Reply:
Hi @pgovind ,
Thanks for the review and guidance on this PR, i will add the changes you specified shortly.
Ramon